core/client/fs/src/main/java/alluxio/client/file/cache/cuckoofilter/ScopeEncoder.java

```diff
@@ -44,6 +44,7 @@ public ScopeEncoder(int bitsPerScope) {
    * @param scopeInfo the scope will be encoded
    * @return the encoded scope
    */
+  @edu.umd.cs.findbugs.annotations.SuppressFBWarnings("RV_RETURN_VALUE_OF_PUTIFABSENT_IGNORED")
```

Comment:
Please use Alluxio's implementation here `core/common/src/main/java/alluxio/annotation/SuppressFBWarnings.java` and remove the added dependency on `edu.umd.cs.findbugs` to avoid licensing issues.

Reply:
done